### ISSUE TITLE
Dependency Issue with latest release - 1.8.6 

### ISSUE THREAD

**Issue by aorfanos1** [elrumordelaluz/reactour#70]
Hi reacttour people, our npm builds have been breaking and it looks like the latest version of your package is having dependency issues. We are getting the following error on build:

`Module not found: Error: Can't resolve 'styled-components' in '/home/jenkins/workspace/{project}/node_modules/reactour/dist/components'`

If this is just me being ignorant I apologize but it looks like rolling back to 1.8.5 resolved the problem before you removed the dependency.

**Comment by elrumordelaluz** [comment]
Hi @aorfanos1, I'm sorry for this.

I didn't documented yet the last change but, basically we switched `styled-components` as a `peerDependency` (so probably you also got an `npm` message when upgrading `reactour` into `1.8.6`).

The change avoids problems with different SC versions in projects that are using it.
Now we [aren't bundling anymore SC](https://www.styled-components.com/docs/faqs#i-am-a-library-author-should-i-bundle-styledcomponents-with-my-library) into `reactour`.

Basically you have to `npm i styled-components` / `yarn add styled-components`.

Let me know if you still having issues in case.

**Comment by aorfanos1** [comment]
Thanks that did it.
